qlib/tests/config.py

```diff
@@ -68,7 +68,7 @@ def get_data_handler_config(
 
 def get_dataset_config(
     dataset_class=DATASET_ALPHA158_CLASS,
-    train=("2008-01-01", "2014-12-31"),
+    train=("2013-01-01", "2014-12-31"),
```

Comment:
Make sure this will not affect others  https://github.com/microsoft/qlib/search?q=get_dataset_config